MapboxNavigation/CarPlayManager.swift

```diff
@@ -329,18 +329,23 @@ extension CarPlayManager: CPInterfaceControllerDelegate {
         }
     }
     public func templateWillDisappear(_ template: CPTemplate, animated: Bool) {
+        typealias TemplateBooleanClosure = (CPTemplate) -> Bool
+        guard let interface = interfaceController else { return }
+        
+        let listOrSearchOrMap = [CPSearchTemplate.self, CPListTemplate.self, CPMapTemplate.self]
```

Comment:
What about CPGridTemplate or CPVoiceControlTemplate, in case the application has a need for one of those?

Reply:
Interesting, but what would we be excluding at that point?

Reply:
Do we need to exclude anything? I’m not sure what the point of this check is in the first place.

Reply:
That makes two of us. Anyway, suggestion applied 👍